homework/list.js

```diff
@@ -1,79 +1,88 @@
 'use strict';
 
-(function() {
+(function(exports) {
+  var ListNoteManager = function() {
+    var _listNoteContent = [];
+    var _wrapper = null;
+  };
 
-  var _listNoteContent = [];
-  var _wrapper = document.querySelector('#note-list-wrapper');
+  ListNoteManager.prototype = {
+    start() {
+        this._wrapper = document.querySelector('#note-list-wrapper');
+        this.fetchList().then((function(data) {
```

Comment:
You should try to compose the promise chain with multiple `then` and `catch`. And you should notice that the `catch` should **log AND re-throw** the exception, to make sure the error is logged in the console and there won't be any exceptions after the error.